bundles/org.openhab.binding.amberelectric/README.md

```diff
@@ -0,0 +1,69 @@
+# Amber Electric Binding
+
+A binding that supports the Australian energy retailer Amber's API (https://www.amber.com.au/) and provides data on the current pricing for buying and selling power, as well as the current level of renewables in the NEM.
+
+## Supported Things
+
+- `service` Amber Electric API 
+
+## Discovery
+
+The binding does not support auto discovery.
```

Comment:
This chapter can be omitted.

Reply:
Sure? Letting the user know it is not supported is also useful info. Other bindings also mention this explicit. Or is there another reason?

Reply:
I'll wait on this. I have the comment about auto discover based on it being in the README for every other binding I've reviewed.

Reply:
I think we usually focus on the available features and functionality, not absence of features. For a pure Internet service binding without any local hardware, I think it's quite obvious that discovery cannot be supported in this case. However, it's minor, and you can decide to either remove it, or leave it as is.

Reply:
I understand what you're saying, but for consistency, I believe it's worth retaining.